packages/synapse-react-client/src/components/SynapseTable/export/ExternalAnalysisPlatformsConstants.tsx

```diff
@@ -15,7 +16,65 @@ type ExternalAnalysisPlatformDescription = {
   PlatformInstructions?: React.ComponentType<PlatformInstructionProps>
 }
 
-export type ExternalAnalysisPlatform = 'cavatica' | 'terra' | 'adworkbench'
+export type ExternalAnalysisPlatform =
+  | 'cavatica'
+  | 'terra'
+  | 'adworkbench'
+  | 'pluto'
+  | 'plutodev'
+
+const plutoAnalysisPlatformDescription: ExternalAnalysisPlatformDescription = {
+  name: 'Pluto',
+  description:
+    'Intuitively store, search, analyze, and visualize complex biological data (including large, raw data files) and run bioinformatics pipelines in a few clicks without needing to write code or manage any of your own cloud infrastructure.',
+  Logo: PlutoLogo,
+  PlatformInstructions: () => (
+    <>
+      <Typography variant="body1" sx={{ fontWeight: 700 }} gutterBottom>
+        You must meet these requirements from Pluto to send data:
+      </Typography>
+      <Typography
+        variant="body1"
+        component={'ol'}
+        gutterBottom
+        sx={{ marginLeft: '10px', li: { mb: 1 } }}
+      >
+        <li>You must complete all required Access Actions in the next step.</li>
+        <li>
+          You must be logged in to a Pluto account.{' '}
+          <Link
+            href={'https://app.pluto.bio/login'}
+            rel={'noopener noreferrer'}
+            target={'_blank'}
+          >
+            Click here to login or register for Pluto
+          </Link>
+          .
+        </li>
+        <li>
+          You must connect your Pluto account to Synapse.{' '}
+          <Link
+            href={'https://help.pluto.bio/en/articles/synapse-pluto'}
+            target="_blank"
+          >
+            Click here for instructions.
+          </Link>
+        </li>
+      </Typography>
+
+      <Typography variant="body1" gutterBottom>
+        Note that we cannot provide support for Pluto. Please contact Pluto's{' '}
+        <Link href="mailto:support@pluto.bio">technical support</Link> for
+        issues related to the above.
+      </Typography>
+    </>
+  ),
+}
+const plutoDevAnalysisPlatformDescription: ExternalAnalysisPlatformDescription =
```

Comment:
Is there a better way that you can think of to support a Dev instance for these integrations (for initial testing)?  I tried to keep them separate, for easy deletion after it's released (on the Pluto side)

Reply:
Nope, I think this approach looks good